quantities/output.py

```diff
@@ -62,6 +62,8 @@
 decaymode_2 = Quantity("decaymode_2")
 gen_match_1 = Quantity("gen_match_1")
 gen_match_2 = Quantity("gen_match_2")
+tau_gen_match_1 = Quantity("tau_gen_match_1")
+tau_gen_match_2 = Quantity("tau_gen_match_2")
```

Comment:
Are still both needed?

Reply:
`gen_match` and `tau_gen_match` are two different things. The second one is for tauSFs, the first one is for the analysis

Reply:
therefore we need both of them